contracts/netmap/contract.go

```diff
@@ -546,6 +546,51 @@ func ListCandidates() iterator.Iterator {
 		storage.ValuesOnly|storage.DeserializeValues)
 }
 
+// IsStorageNode allows to check for the given key presence in the current
+// network map.
+func IsStorageNode(key interop.PublicKey) bool {
+	return IsStorageNodeInEpoch(key, Epoch())
```

Comment:
calling this method does two `Epoch()` calls, do not think it is bad but still

Reply:
It's cheap.